actors/builtin/miner/miner_test.go

```diff
@@ -264,67 +266,71 @@ func TestCommitments(t *testing.T) {
 		deadline := actor.deadline(rt)
 		challengeEpoch := precommitEpoch - 1
 
-		oldSector := actor.commitAndProveSectors(rt, 1, 100, nil)[0]
+		oldSector := actor.commitAndProveSectors(rt, 1, 181, nil)[0]
 
 		// Good commitment.
-		actor.preCommitSector(rt, actor.makePreCommit(101, challengeEpoch, deadline.PeriodEnd(), nil))
+		expiration := deadline.PeriodEnd() + 181*miner.WPoStProvingPeriod
+		actor.preCommitSector(rt, actor.makePreCommit(101, challengeEpoch, expiration, nil))
 
 		// Duplicate pre-commit sector ID
 		rt.ExpectAbort(exitcode.ErrIllegalArgument, func() {
-			actor.preCommitSector(rt, actor.makePreCommit(101, challengeEpoch, deadline.PeriodEnd(), nil))
+			actor.preCommitSector(rt, actor.makePreCommit(101, challengeEpoch, expiration, nil))
 		})
 		rt.Reset()
 
 		// Sector ID already committed
 		rt.ExpectAbort(exitcode.ErrIllegalArgument, func() {
-			actor.preCommitSector(rt, actor.makePreCommit(oldSector.SectorNumber, challengeEpoch, deadline.PeriodEnd(), nil))
+			actor.preCommitSector(rt, actor.makePreCommit(oldSector.SectorNumber, challengeEpoch, expiration, nil))
 		})
 		rt.Reset()
 
 		// Bad sealed CID
 		rt.ExpectAbortConstainsMessage(exitcode.ErrIllegalArgument, "sealed CID had wrong prefix", func() {
-			pc := actor.makePreCommit(101, challengeEpoch, deadline.PeriodEnd(), nil)
+			pc := actor.makePreCommit(102, challengeEpoch, deadline.PeriodEnd(), nil)
```

Comment:
Out of curiosity what motivated the change from 101 to 102?

Reply:
A lot of these failure tests were failing because they were duplicate precommits instead of the intended failure. The change to 102 fixes that.